pkgs/tools/admin/winbox/default.nix

```diff
@@ -65,7 +65,7 @@ symlinkJoin {
     description = "Graphical configuration utility for RouterOS-based devices";
     homepage = "https://mikrotik.com";
     downloadPage = "https://mikrotik.com/download";
-    changelog = "https://wiki.mikrotik.com/wiki/Winbox_changelog";
+    changelog = "https://upgrade.mikrotik.com/routeros/winbox/${version}/CHANGELOG";
```

Comment:
I prefer the old and simple HTML link, this new link forces the browser to download a file.

Reply:
If you find one that's good, but i haven't found anything else with up to date infos.

(The latest changelog in the wiki is for v3.18.)

The new URL is the one beeing used internally by winbox. It's displayed if you click Tools -> Check for Updates. I would therefore expect it to be at least somewhat reliable and stable.